scripts/Dockerfile.build

```diff
@@ -0,0 +1,43 @@
+ARG VERSION
+FROM golang:1.16.5 as builder
+RUN mkdir -p /tmp/dd/datadog-agent
+
+# cache dependencies
+COPY ./scripts/.cache/datadog-agent /tmp/dd/datadog-agent
+WORKDIR /tmp/dd/datadog-agent
+RUN go mod graph \
+| grep -v winutil \
```

Comment:
some package are failing during `go get` 
this is the only way to bypass the error and make the cache usable

Reply:
Would you mind explaining more about why the packages are failing during `go get`?

Reply:
sure, here is an example of an error :
`go: downloading github.com/StackExchange/wmi v1.2.1`
results in 
`github.com/StackExchange/wmi: no Go source files`
this will stop the loop (error code != 0) and the cache will be incomplete